cwms-data-api/build.gradle

```diff
@@ -59,9 +65,11 @@ dependencies {
     }
 
     implementation( 'mil.army.usace.hec:hec-cwms-ratings-io-xml:2.0.2' )
-    implementation( 'mil.army.usace.hec:hec-cwms-ratings-io-jdbc:2.0.2' )
+    implementation( 'mil.army.usace.hec:hec-cwms-ratings-io-jdbc:2.0.2' ){
+        exclude group: "mil.army.usace.hec", module: "cwms-db-jooq-codegen"
+    }
 
-    implementation('org.jooq.pro:jooq:3.11.2') {
+    implementation('org.jooq.pro-java-8:jooq:3.18.7') {
```

Comment:
I had to add cwms-releases and nexus credentials to my build.gradle for it to find the new jooq dependency - @MikeNeilson how does that work at github?  I didn't include my repository changes in this pr b/c I didn't know how that piece was supposed to work.

Reply:
No, we'll need to output those to the maven public.
That was we had done for the previous versions as the way we read the license (this project is a consumer) it was okay.

Though it is one of the things that bothers me.

Reply:
How do I get that artifact pushed into the right repo? Is that something you can do or Darren?

Reply:
Oh I think we had to manually put that there.
@adamkorynta am I remembering that right?

Reply:
They are already there: https://www.hec.usace.army.mil/nexus/#browse/browse:maven-public:org%2Fjooq%2Fpro-java-8

Reply:
well they were supposed to already be there but looks like it only got a version called "local"

Reply:
turned out the maven plugin we use for tagging versions doesn't work with JDK 8 in the latest version. Deploying a new version now.

Reply:
https://www.hec.usace.army.mil/nexus/#browse/browse:maven-public:org%2Fjooq%2Fpro-java-8%2Fjooq%2F3.18.7-jdk8RC01

Reply:
Why is the maven-public artifact version "3.18.7-jdk8RC01" while the cwms-releases on is just 3.18.7?  The java8 part is in the artifact name so it probably doesn't have to be in the version too - right? And why is it RC01 ?

Reply:
The build apparently needs to be able to accesss jooq-parent too. 
```
Execution failed for task ':cwms-data-api:compileJava'.
> Could not resolve all files for configuration ':cwms-data-api:compileClasspath'.
   > Could not resolve org.jooq.pro-java-8:jooq:3.18.7-jdk8RC01.
     Required by:
         project :cwms-data-api
      > Could not resolve org.jooq.pro-java-8:jooq:3.18.7-jdk8RC01.
         > Could not parse POM https://www.hec.usace.army.mil/nexus/repository/maven-public/org/jooq/pro-java-8/jooq/3.18.7-jdk8RC01/jooq-3.18.7-jdk8RC01.pom
            > Could not find org.jooq.pro-java-8:jooq-parent:3.18.7.
              Searched in the following locations:
                - file:/home/runner/.m2/repository/org/jooq/pro-java-8/jooq-parent/3.18.7/jooq-parent-3.18.7.pom
                - https://www.hec.usace.army.mil/nexus/repository/maven-public/org/jooq/pro-java-8/jooq-parent/3.18.7/jooq-parent-3.18.7.pom
                - https://repo.maven.apache.org/maven2/org/jooq/pro-java-8/jooq-parent/3.18.7/jooq-parent-3.18.7.pom
```

Reply:
The one in cwms-releases is the one distributed by jOOQ, where the one in maven-public is has our custom extension. RC01 is because we haven't gotten the artifact up into an application for testing yet and I wasn't sure if the first codegen would work for all of our use-cases. 

You're right about the redundant idk version, no need to repeat ourselves